mm2src/coins/tendermint/tendermint_coin.rs

```diff
@@ -439,6 +450,11 @@ pub enum TendermintInitErrorKind {
     BalanceStreamInitError(String),
     #[display(fmt = "Watcher features can not be used with pubkey-only activation policy.")]
     CantUseWatchersWithPubkeyPolicy,
+    #[display(
+        fmt = "Unable to fetch chain account from WalletConnect. Please try again or reconnect your session - {}",
+        _0
+    )]
+    UnableToFetchChainAccount(String),
```

Comment:
Let's make the error variant generic so it can be used in the future with anything similar to this. When the error originates from WalletConnect, the trace error data will show that.